app/src/main/java/com/willowtree/vocable/MainActivity.kt

```diff
@@ -32,6 +35,8 @@ class MainActivity : BaseActivity() {
         keyboard_button.action = {
             loadFragment(KeyboardFragment())
         }
+
+        presetsViewModel = ViewModelProviders.of(this).get(PresetsViewModel::class.java)
```

Comment:
might run into merge conflicts here, but that's okay :]

Reply:
This was just a placeholder for now. It can be changed if needed.